### ISSUE TITLE
classic

### ISSUE THREAD

**Issue by smmaas** [TheGameCreators/GameGuruRepo#1810]
Can't import gameguru classic levels anymore. (using add existing level)

**Comment by UltraVox001** [comment]
You can not do this. Gameguru Classic maps are not compatible with Max. The two engines are completely different. Max uses Wicked Engine. The "Add Existing Level" function is made to add already existing Max maps in your Project. You can also add the same map in several different Max Projects.

**Comment by smmaas** [comment]
Max would upward compatible with classic and I see no other way to load an classic game

**Comment by UltraVox001** [comment]
To my knowledge, loading a Classic map is done with Gameguru Classic, only. Max is a technically different project. It is the continuity of Classic, but more evolved, and above all, with a different engine. I can't get too far in saying this, and I don't want to mislead you, because I myself haven't used GG Classic for a long time. But I don't think Max will be able to open a Classic map in the near future. But, with TGC, you never know. :) The only things you can use from Classic are Assets and Lua scripts.

**Comment by UltraVox001** [comment]
When in doubt, I gave it a try.
I created a map with GG Classic. I placed one 3D object in front of my feet.
Here is the result in Max :
![Classic](https://user-images.githubusercontent.com/96718175/150641643-5d7764d3-cf1a-4b85-88bf-ca62e800c52c.jpg)
No terrain, no 3D object in front of me.
This 3D object is available for me, in Classic and in Max (it's the same file architecture).
I am very sorry. :(

**Comment by synchromesh62** [comment]
You have in the past been able to import classic maps into Max but not since the new terrain system.
A good question for the stream perhaps 
https://forum.game-guru.com/thread/222405#msg2637129

Here is the big escape in Max 

![BE](https://user-images.githubusercontent.com/35708217/150643581-1bb10326-e321-438b-853e-fe6d30c6b17c.png)

**Comment by plemsoft** [comment]
FYI: For the import to activate the fpm need to be located inside a "real" gameguru classic folder , where the media is also imported from , could you let me know the path you try to import the classic map from ? , thanks for your help :)

**Comment by UltraVox001** [comment]
I followed your advice : I saved a Classic map in the default directory offered by Classic : D:\SteamLibrary\steamapps\common\Game Guru\Files\mapbank
Then I imported it from Max, and I come across a map with Terrain generated by Max, but I don't have any 3D objects on the map. I had submitted 3.

**Comment by plemsoft** [comment]
@UltraVox001 Import will only work if you do it from the "3d Editor" screen using "open" at the moment , will mark it as a bug :)

**Comment by UltraVox001** [comment]
You are right :)
![cm](https://user-images.githubusercontent.com/96718175/150646524-be2388a7-45e7-481a-8bc9-1cb087cc180c.jpg)

**Comment by rvanner** [comment]
@plemsoft Is there any advice that you can add to this thread with regards to importing classic maps?

**Comment by UltraVox001** [comment]
Yes. If GameGuru Classic is only accessible from Steam, you might as well create a new dedicated button. It would directly open the GG Classic maps directory, rather than the Max maps directory.
